Makefile

```diff
@@ -154,7 +154,6 @@ container_build:
 		git config --global --add safe.directory /kepler && make build
 
 build_rpm:
-	rpmautospec generate-changelog packaging/rpm/kepler.spec
```

Comment:
if we remove this, will it fails on RPM build?

Reply:
if we don't need rpmautospec, hence https://github.com/sustainable-computing-io/kepler/blob/main/build/Dockerfile.builder#L33-L34 those lines should be removed as well?

Reply:
i copied CHANGELOG.md to rpm change log

Reply:
> if we don't need rpmautospec, hence https://github.com/sustainable-computing-io/kepler/blob/main/build/Dockerfile.builder#L33-L34 those lines should be removed as well?

good catch, removed